docs/get_started.md

```diff
@@ -116,11 +126,10 @@ the creation of a new management pack project.
      see the [MP Initialization Tool Documentation](references/mp-init.md).
 
 ### Project Templates
-Both templates create a file system that has the basic project structure required to develop and build a Management Pack.
-Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation. `app/adapter.py` is the adapter's
-entry point and the best starting point.
+All the project templates create a file system that has the basic project structure required to develop and build a Management Pack.
+Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation.
 
-The methods inside `adapter.py` are required, and should be modified to generate a custom
+The methods the adapter source code are required, and should be modified to generate a custom
```

Comment:
This sentence needs to be reworked a bit

Reply:
Maybe we even push the descriptions of the methods to a different (language-specific) file, as the signatures are slightly different.